Tweaked vSwarm deployer (#604)
vhive-serverless/invitro: docs/loader.md

```console
# install pre-requisites
sudo apt update
sudo apt -y install git-lfs pip xz-utils
git lfs install
git lfs fetch
git lfs checkout
```
This retrieves the `vSwarm_deploy_metadata.tar.gz` from Git LFS. Then, untar this tarball by running the following command from the root of this repository:

```bash
$ tar -xzvf workloads/container/vSwarm_deploy_metadata.tar.gz -C workloads/container/
```
This untar the tarball into the `workloads/container/yamls` directory, which contains deployment information and deployment YAML files for all vSwarm benchmarks. If you would like to change some of these deployment files of regenerate them, refer to `generate_deploy_info_docs.md` for an outline of what these deployment files are, and how you can regenerate them.

## Single execution

To run load generator use the following command:

```bash
$ go run cmd/loader.go --config cmd/config_knative_trace.json
```

> To run load generator to use vSwarm functions based on `mapper_output.json` run the following:

Reviewer: ```suggestion
To run load generator with vSwarm functions based on `mapper_output.json` run the following:
```

```bash
$ go run cmd/loader.go --config cmd/config_vswarm_trace.json
```

> The difference between instructing the loader to run vSwarm functions is by setting the `VSwarm` flag in the loader configuration to `true`. For information on how to configure the workload for load generator, please refer to `docs/configuration.md`.

Reviewer: ```suggestion
To direct the loader to execute vSwarm functions, set the `VSwarm` flag in the loader configuration `true`. For information on how to configure the workload for load generator, please refer to `docs/configuration.md`.
```
